lib/index.js

```diff
@@ -8,6 +8,7 @@ var Spawn = require("child_process").spawn
   , Ul = require("ul")
   , OneByOne = require("one-by-one")
   , Sliced = require("sliced")
+  , Q = require('q')
```

Comment:
@RubenVerborgh Thanks for suggestion! :cake: 

@krzkaczor I would vote for creating another module for promises support (maybe something like `cobol-promises`) that will have the `cobol` package as dependency. What do you think?